benchmarks/accuracy/README.md

```diff
@@ -0,0 +1,99 @@
+# Accuracy Benchmark
+
+## Overview
+
+The Accuracy Benchmark evaluates the performance of TorchAcc using [FastChat](https://github.com/AlibabaPAI/FastChat_TorchAcc) against a baseline established by Torch native. The benchmark aims to ensure that TorchAcc maintains comparable accuracy levels with Torch native.
+
+## Evaluation Process
+
+To perform the evaluation, follow these steps:
+
+1. Set Baseline
+
+    ```bash
+    bash ./llama.sh <ORIGINAL_MODEL_DIR> 0
```

Comment:
`0, 1` means use torchacc or not?

Reply:
yeah.

```
if [[ $# -ne 2 && $# -ne 3 ]]; then
  echo "Usage: $0 <local_model_dir> <use_torchacc> [checkpiont_output_dir]"
  echo "  local_model_dir: Path to the local directory where the model will be saved."
  echo "  use_torchacc: 0 or 1 to indicate whether to use TorchAcc."
  echo "  checkpoint_output_dir: Optional. Default is the model name in <local_model_dir>."
  exit 1
fi
```

Reply:
I've noticed this code, but users may be confused when reading the README, perhaps it should be explained in the README?

Reply:
Ok, will do.

Reply:
done